force-app/main/default/classes/AccountTriggerTest.cls

```diff
@@ -1,3 +1,10 @@
+/**
+ * @description       :
+ * @author            : ChangeMeIn@UserSettingsUnder.SFDoc
```

Comment:
This part is inserted automatically by Apex Dox, right? I am trying to edit the author but I don't understand where I have to do it. Plus when I try to run 'Apex Dox: Run' I get the following error: ENOENT: no such file or directory, scandir. (I installed the Apex Dox extension). What am I missing? @Szandor72

Reply:
One of the problems is solved:
<img width="620" alt="image" src="https://github.com/iulianachiriac/accounts-discounts/assets/77618420/9c5756b8-b915-4541-919d-2d261ee3da13">

Reply:
> Happy you found the appropriate user setting. The main issue is that this extension is officially not maintained anymore. So even if you fix the second issue, it's a waste of time because DIA needs to move off to another extension
> […](#)
> On Tue, 6 Jun 2023, 20:00 Iuliana Chiriac, ***@***.***> wrote: ***@***.**** commented on this pull request. ------------------------------ In force-app/main/default/classes/AccountTriggerTest.cls <[#1 (comment)](https://github.com/iulianachiriac/accounts-discounts/pull/1#discussion_r1220087183)> : > @@ -1,3 +1,10 @@ +/** + * @description : + * @author : ***@***.*** One of the problems is solved: [image: image] <https://user-images.githubusercontent.com/77618420/243786590-9c5756b8-b915-4541-919d-2d261ee3da13.png> — Reply to this email directly, view it on GitHub <[#1 (comment)](https://github.com/iulianachiriac/accounts-discounts/pull/1#discussion_r1220087183)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AEAGS6QYRSI6OJNXTVRLHLDXJ5V5HANCNFSM6AAAAAAYDN3SDA> . You are receiving this because you were mentioned.Message ID: ***@***.***>

Apex Dox is not maintained anymore? So there would be no point in taking time to solve the second issue. In that case tomorrow I will go forward and create the trigger. Let me know when there is news about the new extension to be used.